modules/extension/complex/src/test/java/org/geotools/filter/expression/FeaturePropertyAccessorTest.java

```diff
@@ -290,6 +290,20 @@ public void testComplexFeature() {
                         "eg:complexAttribute/eg:rootAttribute/eg:multiLeafAttribute",
                         new Hints(FeaturePropertyAccessorFactory.NAMESPACE_CONTEXT, NAMESPACES));
         assertEquals(multiLeafDesc, ex.evaluate(fType));
+
+        // test that executing Java methods is blocked
+        ex =
```

Comment:
(see my other comment on 5seconds)
This MIGHT want to be >5s (build server might be busy)